dist/pages/login.html

```diff
@@ -0,0 +1,102 @@
+            <span class="navbar-toggler-icon"></span>
+          </button>
+          <div class="offcanvas offcanvas-end text-bg-secondary" tabindex="-1" id="offcanvasDarkNavbar" aria-labelledby="offcanvasDarkNavbarLabel">
+            <div class="offcanvas-header">
+              <h5 class="offcanvas-title" id="offcanvasDarkNavbarLabel">Menu</h5>
+              <button type="button" class="btn-close btn-close-white" data-bs-dismiss="offcanvas" aria-label="Close"></button>
+            </div>
+            <div class="offcanvas-body">
+              <ul class="navbar-nav justify-content-end flex-grow-1 pe-3">
+                <li class="mx-auto nav-item">
+                  <form class="d-flex input-group" role="search">
+                    <input class="form-control" type="search" placeholder="Search for friend or posts" aria-label="Search" />
+                    <button class="btn btn-success btn-lg" type="submit">Search</button>
+                  </form>
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link" aria-current="page" href="../../index.html"><i class="fa-solid fa-house"></i> Home</a>
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link" href="/dist/pages/profile.html"><i class="fa-solid fa-user"></i> Profile</a>
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link active" href="/dist/pages/login.html"><i class="fa-solid fa-arrow-right-to-bracket"></i> Login</a>
+                </li>
+              </ul>
+            </div>
+          </div>
+        </div>
+      </nav>
+    </header>
+    <main class="mb-4">
+      <section class="mx-2">
+        <div class="form-login mx-auto my-4 p-4 bg-secondary rounded-3">
+          <div>
+            <form action="profile.html">
+              <div class="login-header">
+                <img class="mb-4 px-4 rounded-circle bg-primary" src="../images/logo/logo_compact_light.png" alt="SoSocial Logo" />
+                <h1 class="mb-3 text-white">
+                  Welcome Back, <br />
+                  Login
+                </h1>
+              </div>
+              <div class="form-floating">
+                <input type="email" class="form-control mb-3" id="floatingInput" placeholder="name@example.com" />
+                <label for="floatingInput">Email address</label>
+              </div>
+              <div class="form-floating">
+                <input type="password" class="form-control mb-3" id="floatingPassword" placeholder="Password" />
+                <label for="floatingPassword">Password</label>
+              </div>
+
+              <div class="checkbox mb-3">
+                <label class="text-white"> <input type="checkbox" class="check-lg" value="remember-me" /> Remember me </label>
+              </div>
+              <button class="w-100 btn btn-lg btn-primary" type="submit">Login in</button>
+            </form>
```

Comment:
You should perform some client-side validation in your form. Things like `required`, `minlength`, `pattern` attributes. See [MDN Docs](https://developer.mozilla.org/en-US/docs/Learn/Forms/Form_validation).

Reply:
I've added required and regex patterns tags to the inputs. I also add min-length to the password, might be redundant as the regex pattern requires 8 characters.

Reply:
Looks great! This is resolved then :-)